media/js/mozorg/contribute/contribute-survey.js

```diff
@@ -2,15 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function($) {
+(function() {
     'use strict';
 
-    $(document).ready(function() {
-        var $survey = $('#survey-message');
-        $survey.addClass('show');
-        setTimeout(function() {
-            $survey.addClass('animate');
-        }, 500);
-    });
-
-})(window.jQuery);
\ No newline at end of file
+    document.getElementById('survey-message').classList.add('show');
```

Comment:
I'm guessing by looking at the old code that this was only hidden initially so it could be animated in. With the animation gone I think we can just show the survey by default even with JS disabled :)

Reply:
Yup you're right this is left-over, removing 👍